### DISCUSSION TITLE
Welcome to wyzeapy Discussions!

### DISCUSSION THREAD

**Discussion by SecKatie** [SecKatie/wyzeapy#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by louisc32578** [comment]
What can I do with WYZE notifications as an entity in HA. Can I turn on V3 sirens via HA? Thanks

**Reply by SecKatie** [reply]
It currently enables or disables the global notifications to your phone
